reactor-core/src/main/java/reactor/core/Exceptions.java

```diff
@@ -298,6 +298,20 @@ public static boolean isMultiple(@Nullable Throwable t) {
 		return t instanceof CompositeException;
 	}
 
+	/**
+	 * Check a {@link Throwable} to see if it is a backtrace, as created by the checkpoint operator or debug utilities.
+	 *
+	 * @param t the {@link Throwable} to check, {@literal null} always yields {@literal false}
+	 * @return true if the Throwable is a backtrace, false otherwise
+	 */
+	public static boolean isBacktrace(@Nullable Throwable t) {
+		if (t == null) {
+			return false;
+		}
+		//FIXME maybe add an interface here for detection purposes
+		return "reactor.core.publisher.FluxOnAssembly.OnAssemblyException".equals(t.getClass().getCanonicalName());
+	}
```

Comment:
I wonder about aligning terminology. The Javadoc for `checkpoint` talks about "assembly tracing" and "assembly traceback" vs the `backtrace` used here. Perhaps `isCheckpointError` or `isCheckpontTraceback"?